src/logging.rs

```diff
@@ -0,0 +1,15 @@
+use log::{info, LevelFilter};
+
+pub fn init_logging(verbose: &String) {
+    let level = match verbose.as_str() {
```

Comment:
We could rename the "verbose" variable and the flags for the commands to "verbosity"